### PR TITLE
add example for 3rdparty configuration

### PR DESCRIPTION
**Pull request by carlaschroder**
Re: https://github.com/owncloud/core/issues/19167

### CODE REVIEW

File: config/config.sample.php [hunk 1]
```diff
@@ -609,7 +609,9 @@
 /**
  * ownCloud uses some 3rd party PHP components to provide certain functionality.
  * These components are shipped as part of the software package and reside in
- * ``owncloud/3rdparty``. Use this option to configure a different location.
+ * ``owncloud/3rdparty``. Use this option to configure a different location. 
+ * For example, if your location is /var/www/owncloud/foo/3rdparty, then the 
+ * correct configuration is '3rdpartyroot' => '/var/www/owncloud/foo/',
```
Comment (nickvergessen) [inline config/config.sample.php]: Actually from reading the code, it looks like it should be including `3rdparty`

Comment (nickvergessen) [inline config/config.sample.php]: Ah no, nevermind

**Comment by nickvergessen** [comment]
:+1:

**Comment by DeepDiver1975** [comment]
:+1:

**Pull request merged**